lib/build.ts

```diff
@@ -166,6 +166,26 @@ async function compile(nodeVersion: string, targetArch: string) {
   return compileOnUnix(nodeVersion, targetArch);
 }
 
+async function hash(filePath: string): Promise<string> {
+  return new Promise<string>((resolve, reject) => {
+    const resultHash = crypto.createHash('sha256');
+    const input = fs.createReadStream(filePath);
+
+    input.on('error', (e) => {
```

Comment:
Would use `once` here
```suggestion
    input.once('error', (e) => {
```

Reply:
We don't want unhandled exceptions. It is more common to use `on`.